.github/workflows/run_tests_prod.yml

```diff
@@ -0,0 +1,15 @@
+name: Run Tests - Prod
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 6,18 * * *'
+
+jobs:
+  sdk-python-tests-prod:
+    uses: Clarifai/clarifai-python/.github/workflows/run_tests.yml@DEVX-662-Periodic-Synthetic-Checks
```

Comment:
Should we need to change this to Master branch once it is merged ?

Reply:
Ohh Yeah, i forgot to change this back. This helps me to test the workflows using current branch. You can leave this comment unresolved. When the PR is Approved, I will resolve this accordingly.